dotnet-authserver/src/TeacherIdentity.AuthServer/Pages/SignIn/Register/ExistingAccountPhoneConfirmation.cshtml.cs

```diff
@@ -25,6 +25,7 @@ public class ExistingAccountPhoneConfirmation : BasePhoneConfirmationPageModel
     }
 
     public override string? MobileNumber => HttpContext.GetAuthenticationState().ExistingAccountMobileNumber;
```

Comment:
I guess technically, this comes from the normalised mobile number column so shouldn't really need parsing again. No harm in being safe though

Reply:
This one comes from the un-normalised value but it is assumed to be valid. As this isn't in production yet I figured there's no way an invalid value could ever be observed here.

Reply:
Ah ok, originally was coming from the mobile_number column which was assumed to be normalised. So now it's from the same column but normalisation isn't a guarantee. Got it!